templates/navigation/oe-corporate-blocks-eu-footer--standardised.html.twig

```diff
@@ -0,0 +1,87 @@
+{#
+/**
+ * @file
+ * Theme override for the corporate European Union footer block.
+ *
+ * Available variables:
+ * - site_owner: Site's owner.
+ * - site_specific_footer: renderable array for site specific footer links
+ * - corporate_footer: renderable array for corporate footer links
+ * @see ./module/contrib/oe_corporate_blocks/templates/oe-corporate-blocks-eu-footer.html.twig
+ */
+#}
+
+{% set _logo_path = ecl_logo_path ~ '/logo--' ~ current_language_id|to_internal_language_id ~ '.svg' %}
+
+{% set _sections = [
+  {
+    'title': {
+      'link': {
+        'label': site_specific_footer.site_identity,
+        'path': url('<front>'),
+      }
+    },
+    'description': site_owner ? 'This site is managed by the @name'|t({'@name': site_owner}),
+    'section_id': 1,
+  },
+  [
+    {
+      'section_id': 2,
+      'title_class_name': 'ecl-footer-standardised__title--separator',
+    },
+    {
+      'section_id': 2,
+      'title_class_name': 'ecl-footer-standardised__title--separator',
+    }
+  ],
+  {
+    'section_id': 3,
+    'title_class_name': 'ecl-footer-standardised__title--separator',
+  },
+  {
+    'section_id': 6,
+  },
+  {
+    'logo': {
+      'path': 'https://europa.eu/',
+      'language': current_language_id,
+      'src_mobile': _logo_path,
+      'src_desktop': _logo_path,
+    },
+    'description': "Discover more on <a href=':path' class='ecl-link ecl-link--standalone'>@label</a>"|t({
+      ':path': 'https://europa.eu/',
+      '@label': 'europa.eu'
+    }),
```

Comment:
If possible, let's make sure we keep ECL classes outside the translatable string. Since that link will not change according to language (at least we assume so), we can pass the entire link to the t() function.

Reply:
done as we agreed